baseapp/grpcserver.go

```diff
@@ -80,7 +88,25 @@ func (app *BaseApp) RegisterGRPCServer(server gogogrpc.Server) {
 			}
 		}()
 
-		return handler(grpcCtx, req)
+		// Extract the existing span context from the incoming request
+		parentCtx := otel.GetTextMapPropagator().Extract(grpcCtx, propagation.HeaderCarrier(md))
+
+		// Start a new span representing the request
+		// The span ends when the request is complete
+		grpcCtx, span := tracer.Start(parentCtx, grpcInfo.FullMethod, trace.WithSpanKind(trace.SpanKindServer))
+		defer span.End()
+
+		span.SetAttributes(attribute.String("http.method", grpcInfo.FullMethod))
```

Comment:
Why do you add the `http.method` attribute?  A span for `FullMethod` is created in the stats handler already